components/input/hotkey.cpp

```diff
@@ -53,142 +53,142 @@ namespace Input
 
         switch (key)
         {
-            case Input::KEY_F1:
+            case (int)Input::Key::KEY_F1:
                 hotkeyName += "F1";
                 break;
-            case Input::KEY_F2:
+            case (int)Input::Key::KEY_F2:
                 hotkeyName += "F2";
                 break;
-            case Input::KEY_F3:
+            case (int)Input::Key::KEY_F3:
                 hotkeyName += "F3";
                 break;
-            case Input::KEY_F4:
+            case (int)Input::Key::KEY_F4:
                 hotkeyName += "F4";
                 break;
-            case Input::KEY_F5:
+            case (int)Input::Key::KEY_F5:
                 hotkeyName += "F5";
                 break;
-            case Input::KEY_F6:
+            case (int)Input::Key::KEY_F6:
                 hotkeyName += "F6";
                 break;
-            case Input::KEY_F7:
+            case (int)Input::Key::KEY_F7:
                 hotkeyName += "F7";
                 break;
-            case Input::KEY_F8:
+            case (int)Input::Key::KEY_F8:
                 hotkeyName += "F8";
                 break;
-            case Input::KEY_F9:
+            case (int)Input::Key::KEY_F9:
                 hotkeyName += "F9";
                 break;
-            case Input::KEY_F10:
+            case (int)Input::Key::KEY_F10:
                 hotkeyName += "F10";
                 break;
-            case Input::KEY_F11:
+            case (int)Input::Key::KEY_F11:
                 hotkeyName += "F11";
                 break;
-            case Input::KEY_F12:
+            case (int)Input::Key::KEY_F12:
                 hotkeyName += "F12";
                 break;
-            case Input::KEY_F13:
+            case (int)Input::Key::KEY_F13:
                 hotkeyName += "F13";
                 break;
-            case Input::KEY_F14:
+            case (int)Input::Key::KEY_F14:
                 hotkeyName += "F14";
                 break;
-            case Input::KEY_F15:
+            case (int)Input::Key::KEY_F15:
```

Comment:
Casting to int is done because `key` is of type int. But still it would be better to cast `key` to `enum` inside `switch` one time than to apply cast to each of the cases.

Reply:
Ok, I'll do it